contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/statsbeat.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import json
+import os
+import platform
+
+import requests
+
+from opencensus.ext.azure.common.utils import azure_monitor_context
+from opencensus.metrics.export.gauge import LongGauge
+from opencensus.metrics.label_key import LabelKey
+from opencensus.metrics.label_value import LabelValue
+
+_AIMS_URI = "http://169.254.169.254/metadata/instance/compute"
+_AIMS_API_VERSION = "api-version=2017-12-01"
+_AIMS_FORMAT = "format=json"
+
+_STATS_CONNECTION_STRING = "InstrumentationKey=c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint=https://dc.services.visualstudio.com/"  # noqa: E501
+
+_TELEMETRY_NAME = "Statsbeat"
+_ATTACH_METRIC_NAME = "Attach"
+
+_RP_NAMES = ["aks", "appsvc", "function", "vm", "unknown"]
```

Comment:
Is there a plan to determine if SDK is running in AKS?

Reply:
AKS is not supported yet.. since there is rpId is indeterministic.

Reply:
WIll remove.